guides/wide-ep-lws/manifests/modelserver/base/decode.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           - name: dshm
             emptyDir:
               medium: Memory
-              sizeLimit: 3Gi # roughly 32MB per local DP plus scratch space
+              sizeLimit: 1Gi # roughly 32MB per local DP plus scratch space
```

Comment:
I was hitting OOMs testing some variation of this with 1Gi of shm. I forget what feature it was at this point just wanted to call it out